spec/ruby/core/warning/element_reference_spec.rb

```diff
@@ -8,6 +8,14 @@
     end
   end
 
+  ruby_version_is '3.3' do
+    it "returns default values for :performance category" do
+      ruby_exe('p Warning[:performance]').chomp.should == "false"
+      ruby_exe('p Warning[:performance]', options: "-w").chomp.should == "false"
+      ruby_exe('p Warning[:performance]', options: "-W").chomp.should == "false"
```

Comment:
`-W` is the same as `-w` so I'd test just `-w`, because subprocess tests like this are not so cheap.